src/xtgeo/well/wells.py

```diff
@@ -3,46 +3,82 @@
 """Wells module, which has the Wells class (collection of Well objects)"""
 
 
+import functools
+import warnings
 from distutils.version import StrictVersion
+from typing import List
 
+import deprecation
 import pandas as pd
 
 import xtgeo
 
 from . import _wells_utils
+from .well1 import Well
 
 xtg = xtgeo.common.XTGeoDialog()
 logger = xtg.functionlogger(__name__)
 
 
-class Wells(object):
+def wells_from_files(filelist, *args, **kwargs):
+
+    """Import wells from a list of files (filelist).
+
+    Creates a Wells object from a list of filenames. Remaining arguments are
+        the same as :func:`xtgeo.well_from_file`.
+
+    Args:
+        filelist (list of filenames): List with file names
+
+    Example:
+        Here the from_file method is used to initiate the object
+        directly::
+
+            >>> mywells = Wells(
+            ...     [well_dir + '/OP_1.w', well_dir + '/OP_2.w']
```

Comment:
Is this example still valid since `xtgeo.well_from_file` returns a `Well`-object?